### PR TITLE
Refactoring: Too long lines, missing whitespaces

### PR DESCRIPTION
**Pull request by ari3s**
Fix of too long lines (longer than 120 characters) and missing whitespaces.

Signed-off-by: Jan Beran <jberan@redhat.com>

### CODE REVIEW

**Comment by mkutlak** [comment]
Looks like the PR https://github.com/abrt/retrace-server/pull/212 caused a merge conflict.

**Comment by ari3s** [comment]
@mkutlak Thanks for the review, I'll fix the conflict.

**Comment by xsuchy** [comment]
retest this please

File: src/manager.wsgi [hunk 1]
```diff
@@ -373,16 +381,17 @@ def application(environ, start_response):
                                   "<tr><td colspan=\"2\">%s <code>retrace-server-interact %s shell</code></td></tr>" \
                                   "<tr><td colspan=\"2\">%s <code>retrace-server-interact %s %s</code></td></tr>" \
                                   "<tr><td colspan=\"2\">%s <code>man retrace-server-interact</code> %s</td></tr>" \
-                                  % (_("This is an interactive task"), _("You can jump to the chrooted shell with:"), filename,
-                                     _("You can jump directly to the debugger with:"), filename, debugger,
-                                     _("see"), _("for further information about cmdline flags"))
+                                  % (_("This is an interactive task"), _("You can jump to the chrooted shell with:"), \
```
Comment (mkutlak) [inline src/manager.wsgi]: The `\` at the end of the lines are not needed because everything is in brackets. The indentation should be left unchanged.

**Comment by mkutlak** [comment]
```
pylint-3 --rcfile=pylintrc $(find ./src/ -regextype posix-extended -regex '.*(retrace-server[a-z-]+|(.(py(.in)?|wsgi)))') src/coredump2packages 

************* Module manager (src/manager.wsgi)                                                                                                                                                                                       
C:129, 0: Line too long (125/120) (line-too-long)                                                                                                                                                                  
C:154, 0: Line too long (128/120) (line-too-long)                                                                                                                                                                  
C:185, 0: Line too long (129/120) (line-too-long)                                                                                                                                                                  
C:219, 0: Line too long (126/120) (line-too-long)
```

These lines are still too long.

**Pull request merged**